src/http/index.ts

```diff
@@ -25,11 +25,11 @@ export const start = (
     app.use(cors())
 
     // Rest endpoints
-    app.use('/api/v1', dataQueryEndpoints(networkNode, streamFetcher, metricsContext))
+    app.use('/api/v1', dataQueryEndpoints(cassandraStorage!, streamFetcher, metricsContext))
```

Comment:
I'm assuming these `cassandraStorage!` statements will be covered by an `if`-statement later on? I.e. `if (cassandraStorage !== undefined)`

Reply:
Would it make sense to put that validation to config level? We'd check that if isStorageNode is true, there must be all required configurations for storage node (`cassandraStorage`, `storageConfig` etc.). And we could make a type definition `BrokerConfig|StorageNodeConfig`.

Reply:
Makes sense, a brings us one step closer to a potential separation between broker / storage.

Reply:
Implemented here: https://github.com/streamr-dev/broker/pull/254